README.md

```diff
@@ -1,2 +1,4 @@
 # longhorn-tests
-Various test cases for longhorn
+A setup and a teardown script for setting up machines for various longhorn tests
```

Comment:
I will move them to its own folder and add its own readme/usage. Talked to Bill, we need to run setup/teardown in a container, there is a docker daemon in that VM(I didn't use sudo). That could take me a day or two I guess.